components/proxy/src/test/java/com/hotels/styx/proxy/BrotliTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+  Copyright (C) 2013-2024 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx.proxy;
+
+import io.netty.handler.codec.compression.Brotli;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class BrotliTest {
+    @Test
+    @EnabledOnOs(value = { OS.LINUX, OS.MAC })
+    public void brotliCompressionIsAvailable() throws Throwable {
+        Brotli.ensureAvailability();
```

Comment:
You can remove this because the below `assert` will fail if the native library is not available.

Reply:
👍   I just pushed an update.  I removed BrotliTest.java and moved 1 assertion into HttpCompressorTest